config/constants.js

```diff
@@ -16,4 +16,15 @@ const consoleFontColors = {
   RED: "\x1b[31m",
 };
 
-export { statusCodes, errorMessages, consoleFontColors, userMessages };
+const dateConstants = {
+  YYYY_MM_DD: "yyyy-MM-dd",
```

Comment:
rename it please
if you will change data format then you will need to rename constant